padinfo/padinfo.py

```diff
@@ -1409,3 +1410,20 @@ async def expcurve(self, ctx, start, end: Optional[int], *, query=''):
         state = ClosableEmbedViewState(original_author_id, ClosableEmbedMenu.MENU_TYPE, query,
                                        query_settings, ExperienceCurveView.VIEW_TYPE, props)
         await menu.create(ctx, state)
+
+    @commands.command()
+    async def board(self, ctx, *, message):
+        """Generate a Dawnglare link from the user provided string.
+        Rows/columns separated with spaces.
+        Default fill direction is right then down.
+        Use -1 to invert fill, going down then right."""
+
+        board = BoardGenerator(message)
+
+        if board.invalid_board:
+            await send_cancellation_message(ctx, "An invalid board was defined. Please enter a 5x4, 6x5, or 7x6 "
+                                                 "board. Rows/columns must be separated with spaces.")
+        elif board.invalid_orbs:
+            await send_cancellation_message(ctx, f"An invalid letter was used. Only {board.allowed_letters}are allowed.")
```

Comment:
If someone does an invalid board AND has invalid letters, you probably wanna return both errors. So this shouldn't be an elif here. Then you can send the link if there were no errors